HW04/Pavel_Karenda/spec/my_map_spec.rb

```diff
@@ -0,0 +1,25 @@
+require_relative '../my_map'
+require_relative '../my_each'
+
+class Array
+  include CustomMap
+  include CustomEach
+end
+
+RSpec.describe Array do
+  subject { described_class.new([0, 1, 2, 3]) }
+
+  describe '#my_each' do
+    context 'when no block is given' do
+      it 'returns start array' do
+        expect(subject.my_map.to_a).to eq subject
+      end
+    end
+
+    context 'when block is given' do
+      it 'returns modified array' do
+        expect(subject.my_map { |element| p element + 2 }).to eq([2, 3, 4, 5])
```

Comment:
Why do you need `p` here?
```suggestion
        expect(subject.my_map { |element| element + 2 }).to eq([2, 3, 4, 5])
```

Reply:
is was added to have something in stdout. Deleted.